src/main/java/com/sandvoxel/generitech/common/blocks/Blocks.java

```diff
@@ -59,7 +59,7 @@
     BLOCK_PULVERIZER(BlockPulverizer.class, ItemPulverizer.class),
 
     BLOCK_MACHINEMATRICS(BlockMachineMatrics.class, ItemMachineMatrics.class),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "Blocks" constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUnusedPrivateMethod)